src/content/docs/tutorial-reporting-data/capture-the-right-data.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Capture the right data with New Relic" 
+metaDescription: "This guide helps you set up the best telemetry possible to describe the runtime operation of your services."
+redirects:
+  - /docs/apm/agents/manage-apm-agents/agent-data/custom-instrumentation
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/service-characterization-optimize-telemetry-data-guide/
+---
+
+Modern monitoring systems can give you important insight into the details of a service when they report the right data. Distributed trace, byte code, or script instrumentation allows teams to quickly collect detailed service telemetry. Unfortunately, operations teams usually aren't in the best position to evaluate the quality of the telemetry they get.
+
+Revealing improperly instrumented services to users creates a situation that can put customer satisfaction at risk. This burn-in period is difficult to escape as new features can come from code bases without a strong link between software delivery and observability programs. Service instrumentation planning is the approach used to describe a single service runtime through telemetry, and this guide focuses on the metrics of your application's code as well as external measurements through synthetic testing. 
```

Comment:
Not sure what "burn-in" period means in this context.

Reply:
I removed it and reworded this, hopefully it's more clear?